x/migration/keeper/msg_server_claim_morse_supplier.go

```diff
@@ -60,6 +82,7 @@ func (k msgServer) ClaimMorseSupplier(
 
 	// Ensure that the given MorseClaimableAccount has not already been claimed.
 	if morseClaimableAccount.IsClaimed() {
+		isAlreadyClaimed = true
```

Comment:
What's this for?

Reply:
I #PUC'd the comment above its declaration; it's needed to track whether the Morse account was already claimed prior to the execution of the rest of the message handler. 

We want to restore the normal gas behavior as much as possible. In this case, that means gas WILL be charged for a **valid** claim message which has already been processed. This mitigates a replay attack on Morse claim messsages, for example.